code/game/mecha/working/ripley.dm

```diff
@@ -56,33 +60,68 @@
 		else
 			add_overlay(occupant ? "ripley-g-full" : "ripley-g-full-open")
 
+/obj/mecha/working/ripley/check_for_internal_damage(list/possible_int_damage,ignore_threshold=null)
+	if (!enclosed)
+		possible_int_damage -= MECHA_INT_TEMP_CONTROL //if we don't even have an air tank, these two doesn't make a ton of sense.
+		possible_int_damage -= MECHA_INT_TANK_BREACH

```

Comment:
`possible_int_damage -= (MECHA_INT_TEMP_CONTROL + MECHA_INT_TANK_BREACH)`
??

Reply:
The internal tank breach is impossible if the MK-I doesn't have a tank at all, and the internal fire didn't make mush sense when you should be able to see the fire and put it out externally. This removes them from the pool of possible events.

Reply:
sorry for not being clear, my concern is that you're doing 2 expressions when it can be shortened to one.

Reply:
Oh, gotcha. I'll make the change